tests/unit/config/param_test.c

```diff
@@ -33,6 +33,9 @@ struct test_params {
     uint64_t test16;
 
     uint32_t test17;
+
+    double   test18;
```

Comment:
I would prefer moving this guy prior to `test11` and after `test10`. and renaming all the rest, but it is a little tedious.

Reply:
Named them after the type they test.